src/rgw/rgw_auth_s3.cc

```diff
@@ -29,8 +29,6 @@ static const char *signed_subresources[] = {
   "torrent",
   "uploadId",
   "uploads",
-  "start-date",
```

Comment:
@oritwas 
no , start-date and end-date is only parameters， should not be treated as sub-resource ， I introduced this mistake at last commit, so I think it need wipe out.
In fact I think usage should really be put here, what`s your opinion?